boards/arm/96b_ivy5661/support/openocd.cfg

```diff
@@ -0,0 +1,12 @@
+source [find board/stm32f4discovery.cfg]
```

Comment:
Is this correct, saying the board cfg is stm32f4discovery.cfg seems wrong

Reply:
Yes, this it not correct. 
UWP5661 does just support J-Link for debugging. We are still looking for the root cause that how to support OpenOCD as well.
May I remove this config for this commit, and support OpenOCD in the future.